static/js/zamboni/files.js

```diff
@@ -330,7 +330,10 @@ function bind_viewer(nodes) {
         this.top = null;
         this.last = null;
         this.compute = function(node) {
-            Highlighter.highlight(node.find('#diff, #content'));
+            let content = node.find('#diff, #content');
+            if (content.length) {
+                Highlighter.highlight(content);
+            }
```

Comment:
Bonus: This fixes a JavaScript error in `highlight.js` when accessing binary files.